op-challenger/game/fault/types/types.go

```diff
@@ -108,6 +109,42 @@ func (t TraceType) GameType() GameType {
 	}
 }
 
+type ServerType string
+
+const (
+	ServerTypeOpProgram ServerType = "op_program"
+	ServerTypeKona      ServerType = "kona"
+)
+
+var ServerTypes = []ServerType{ServerTypeOpProgram, ServerTypeKona}
+
+func (t ServerType) String() string {
+	return string(t)
+}
+
+// Set implements the Set method required by the [cli.Generic] interface.
+func (t *ServerType) Set(value string) error {
+	if !ValidServerType(ServerType(value)) {
+		return fmt.Errorf("unknown server type: %q", value)
+	}
+	*t = ServerType(value)
+	return nil
+}
```

Comment:
This talks about the CLI but doesn't seem to actually by used by the CLI anywhere.